docs/tools/report-converter.md

```diff
@@ -186,6 +188,19 @@ clang++ -fsanitize=thread -g tsan.cpp
 report-converter -t tsan -o ./tsan_results tsan.output
 ```
 
+### [Leak Sanitizer](https://clang.llvm.org/docs/LeakSanitizer.html)
+- Compile with `-g` and `-fsanitize=address`  to get proper debug information in your binary.
+```sh
+# Compile your program.
+clang -fsanitize=address -g lsan.c
+
+# Run your program and redirect the output to a file.
+ASAN_OPTIONS=detect_leaks=1 ./a.out > lsan.output 2>&1
+
+# Generate plist files from the output.
+report-converter -t lsan ./lsan_results lsan.output
```

Comment:
lost a "-o" option before "./lsan_results"

Reply:
Oops! Let me fix that in a pull request! 

Thanks for pointing it out 😄

Reply:
You are welcome, and in addition , could you please help me to check issue #3373 if you have extra time,
I compiled the newest code this morning ,but I can not convert lsan report, even use the official leak sanitizer output, I am really confused now , I don't know which of my steps is wrong or maybe I lost something😥.